README.md

```diff
@@ -35,7 +47,7 @@ The bot listen to the 8181 port so you should redirect to this one in you NAT co
 
 2.TODO list
 
-### TODOLIST
+### TODOLIST (outdated!)
```

Comment:
To make merging into upstream easier, I'd put this at the top of the file. (e.g. `Hereafter the updated readme from the original repo (N.B. the TODOLIST is outdated).`)